spec/index.bs

```diff
@@ -6475,8 +6475,24 @@ following members:
 
                                     :  {{GPUBindGroupLayoutEntry/externalTexture}}
                                     ::
-                                        - |resource| is a {{GPUExternalTexture}}.
+                                        - |resource| is either a {{GPUExternalTexture}} or a {{GPUTextureView}}.
                                         - |resource| is [$valid to use with$] |this|.
+                                        - If |resource| is a
+                                            <dl class=switch>
+                                                : {{GPUTextureView}}
+                                                ::
+                                                    - |resource|.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/usage}}
+                                                        includes {{GPUTextureUsage/TEXTURE_BINDING}}.
```

Comment:
nit: for consistency with the other bullets here, even though this whole block is already inconsistent about "is" vs "must be" style:

```suggestion
                                                        must include {{GPUTextureUsage/TEXTURE_BINDING}}.
```

Reply:
Done